lgt8f/boards.txt

```diff
@@ -3,7 +3,7 @@
 
 # Menu options
 menu.clock_source=Clock Source
-menu.clock=Clock
+menu.clock_div=Clock Divder
```

Comment:
Please correct it:

`Clock Divder` -> `Clock Divider`

Reply:
Fixed